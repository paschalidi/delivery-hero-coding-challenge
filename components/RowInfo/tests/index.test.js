/* eslint-env jest */

import { shallow } from 'enzyme';
import React from 'react';

import Component from '../index';


describe('Snapshot Testing', () => {
  it('RowInfo', () => {
    const component = shallow(<Component />);
    expect(component).toMatchSnapshot();
  });
});
