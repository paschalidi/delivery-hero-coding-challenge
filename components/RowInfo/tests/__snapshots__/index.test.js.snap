// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Testing RowInfo 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <RowInfo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <style>
          
              .flex--center {
                  flex: 1;
                  display: flex;
                  align-items: center;
                  justify-content: center;
                  width: 100vw;
                  padding: 2vw 0;
                  border-bottom: 1px solid #f8f8f8;
              }
          
        </style>,
        <div
          className="text--md"
        >
           Restaurants in total
        </div>,
      ],
      "className": "background-color--white flex--center info",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "
              .flex--center {
                  flex: 1;
                  display: flex;
                  align-items: center;
                  justify-content: center;
                  width: 100vw;
                  padding: 2vw 0;
                  border-bottom: 1px solid #f8f8f8;
              }
          ",
        },
        "ref": null,
        "rendered": "
              .flex--center {
                  flex: 1;
                  display: flex;
                  align-items: center;
                  justify-content: center;
                  width: 100vw;
                  padding: 2vw 0;
                  border-bottom: 1px solid #f8f8f8;
              }
          ",
        "type": "style",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            undefined,
            " Restaurants in total",
          ],
          "className": "text--md",
        },
        "ref": null,
        "rendered": Array [
          undefined,
          " Restaurants in total",
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <style>
            
              .flex--center {
                  flex: 1;
                  display: flex;
                  align-items: center;
                  justify-content: center;
                  width: 100vw;
                  padding: 2vw 0;
                  border-bottom: 1px solid #f8f8f8;
              }
          
          </style>,
          <div
            className="text--md"
          >
             Restaurants in total
          </div>,
        ],
        "className": "background-color--white flex--center info",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "
              .flex--center {
                  flex: 1;
                  display: flex;
                  align-items: center;
                  justify-content: center;
                  width: 100vw;
                  padding: 2vw 0;
                  border-bottom: 1px solid #f8f8f8;
              }
          ",
          },
          "ref": null,
          "rendered": "
              .flex--center {
                  flex: 1;
                  display: flex;
                  align-items: center;
                  justify-content: center;
                  width: 100vw;
                  padding: 2vw 0;
                  border-bottom: 1px solid #f8f8f8;
              }
          ",
          "type": "style",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              undefined,
              " Restaurants in total",
            ],
            "className": "text--md",
          },
          "ref": null,
          "rendered": Array [
            undefined,
            " Restaurants in total",
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
